ape_fantom/ecosystem.py

```diff
@@ -38,3 +58,63 @@ class Fantom(Ethereum):
     @property
     def config(self) -> FantomConfig:  # type: ignore[override]
         return cast(FantomConfig, self.config_manager.get_config("fantom"))
+
+    def create_transaction(self, **kwargs) -> TransactionAPI:
+        """
```

Comment:
I dont think we need this change for `ape-fantom`. I am pretty sure Fantom supports EIP-1559 transactions, so the default should be dyanamic so you dont need to re-implement this method; we can use the one from the base class from `ape_ethereum`

https://fantom.foundation/blog/understanding-fantom-transaction-fees/#:~:text=To%20mitigate%20gas%20price%20volatility,specific%20version%20of%20EIP%2D1559.

or no?